src/select-menu/src/SelectMenu.js

```diff
@@ -186,7 +188,12 @@ SelectMenu.propTypes = {
   /**
    * The height of the items in the select menu list
    */
-  itemHeight: PropTypes.number
+  itemHeight: PropTypes.number,
+
+  /**
+   * When true, menu auto focuses on the search/filter bar.
+   */
+  shouldAutoFocus: PropTypes.bool
```

Comment:
Can we add this to the `SelectMenuProps` interface in `index.d.ts` as well? (Listing `@default true` might be helpful as well)